app/src/main/kotlin/com/wire/android/ui/calling/common/CallerDetails.kt

```diff
@@ -38,6 +39,7 @@ fun CallerDetails(
     avatarAssetId: ImageAsset.UserAvatarAsset?,
     conversationType: ConversationType,
     membership: Membership,
+    isCallingLabel: String
```

Comment:
The prefix is `isSomething` to me at least indicates that the value is a Boolean,  maybe we can rename it to `callingLabel`.